Firestore/Source/Remote/FSTRemoteStore.mm

```diff
@@ -497,13 +495,13 @@ - (void)addBatchToWritePipeline:(FSTMutationBatch *)batch {
 
   if ([self shouldStartWriteStream]) {
     [self startWriteStream];
-  } else if ([self isNetworkEnabled] && self.writeStream.handshakeComplete) {
-    [self.writeStream writeMutations:batch.mutations];
+  } else if ([self isNetworkEnabled] && _writeStream->handshake_complete()) {
```

Comment:
I think on the other platforms this is now _writeStream->is_open() && _writeStream->handshake_complete().